### ISSUE TITLE
day/night cycle

### ISSUE THREAD

**Issue by Planutim** [skycoin/cx-game#451]
-Day/night cycle

-Time for cycle

-where in cycle we are from 0 to 1

**Comment by ted537** [comment]
Is "lighting palette texture" a different concept from the lighting currently present in the game?

**Comment by Planutim** [comment]
yes, having color tints for different day cycles is the idea here
